src/Compilers/CSharp/Portable/Lowering/Instrumentation/DynamicAnalysisInjector.cs

```diff
@@ -421,10 +498,15 @@ private static SyntaxNode SyntaxForSpan(BoundStatement statement)
 
             return syntaxForSpan;
         }
-        
-        private static MethodSymbol GetCreatePayload(CSharpCompilation compilation, SyntaxNode syntax, DiagnosticBag diagnostics)
```

Comment:
>GetCreatePayload [](start = 36, length = 16)

It looks like the only difference between the two methods that replaced this one is that they are using different WellKnownMember values. Consider passing that value as a parameter instead of splitting the function in two.